### PR TITLE
Priority label

### PR DESCRIPTION
**Pull request by saifudinbadani**
Please review my code for priority and label component and logic

### CODE REVIEW

File: src/components/HomePage/MainContent.js [hunk 1]
```diff
@@ -8,6 +8,9 @@ import './Homepage.css';
 export const MainContent = ({setModalOpen}) => {
     const [title, setTitle ] = useState('');
     const [content, setContent ] = useState('');
+    const [label, setLabel] = useState('');
+    const [priority, setPriority] = useState('low');
+
```
Comment (devsurajrai) [inline src/components/HomePage/MainContent.js:13]: when you get to use more than 2 usestates, consider using reducer for the same.

**Review by devsurajrai — APPROVED** [review]
Everything looks good. Just try to use reducer when using multiple states related to same component.

File: src/components/HomePage/Homepage.css [hunk 2]
```diff
@@ -60,16 +67,37 @@
...
+.badge-note{
+    top: 0.4rem;
+    right: 0.4rem;
+    background-color: var(--blue-100)
+}
 
+.priority-note{
+    bottom: 0.4rem;
+    left: 0.4rem;
+    background-color: var(--blue-300);
```
Comment (palash23jain) [inline src/components/HomePage/Homepage.css:99]: good use of CSS variables

**Review by palash23jain — APPROVED** [review]
clean code, looks good to me

File: src/components/HomePage/MainContent.js (hunk 1 above)
Comment (saifudinbadani) [reply src/components/HomePage/MainContent.js:13]: Yes, I will do this when refactoring

**Pull request merged**